src/main/java/layron/tms/controller/TaskController.java

```diff
@@ -0,0 +1,59 @@
+package layron.tms.controller;
+
+import java.util.List;
+import layron.tms.dto.task.CreateTaskRequestDto;
+import layron.tms.dto.task.TaskDto;
+import layron.tms.dto.task.UpdateTaskRequestDto;
+import layron.tms.service.task.TaskService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api")
+public class TaskController {
+    private final TaskService taskService;
+
+    @PostMapping("/{projectId}/tasks")

```

Comment:
For me personally, it seems more logical to have projects/{projectId}/tasks URL in order to show that task belongs to project (in every endpoint)